### PR TITLE
adding a rock paper scissor game

### LINKED ISSUE #3905: [New game]: Stone paper Scissor Game
### 🎮 Game Request

Game logic and basic description
simple js,html,css web app for stone paper scissor game based on simple logics.

### Point down the features

Game points
Screenshot 2024-01-04 173544

### Select program in which you are contributing

_No response_

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Vaishnavi1047**
## PR Description 📜
Adding Stone Paper Scissor
Fixes #3905 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸
Screenshot 2024-06-07 202427

--- 
<br>
@kunjgit pls look into this

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Vaishnavi1047 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Vaishnavi1047,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Comment by Vaishnavi1047** [comment]
ok sir

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Vaishnavi1047,
You need to make the following changes:
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

**Comment by Vaishnavi1047** [comment]
Update the readme like by adding what? It already have the info na?

**Comment by Vaishnavi1047** [comment]
and pls add the gssoc label in this issue

File: assets/images/rock-2004.png
Comment (ishita-43) [inline assets/images/rock-2004.png:1]: rename image as name same as name of your game

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Review by ishita-43 — APPROVED** [review]
Good to merge

**Review by kunjgit — APPROVED** [review]

**Comment by kunjgit** [comment]
Thank you  @Vaishnavi1047 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**